spec/erb_lint/linters/comment_syntax_spec.rb

```diff
@@ -52,15 +52,25 @@
     let(:file) { <<~FILE }
       <% # first bad comment %>
       <%= # second bad comment %>
+      <%- # third bad comment %>
     FILE
 
-    it "reports two offenses" do
-      expect(subject.size).to(eq(2))
+    it "reports all offenses" do
+      expect(subject.size).to(eq(file.each_line.count))
     end
 
     it "reports the suggested fixes" do
-      expect(subject.first.message).to(include("Bad ERB comment syntax. Should be <%# without a space between."))
-      expect(subject.last.message).to(include("Bad ERB comment syntax. Should be <%#= without a space between."))
```

Comment:
I almost switched this to `<%#`, but I realized one might just be trying to temporarily comment something out.

```diff
 blah blah
-<%= debugging_something %>
+<%#= debugging_something %>
 blah
```